app/src/main/java/inz/MainActivity.java

```diff
@@ -117,4 +135,24 @@ private void getFile(Uri pickerInitialUri) {
 
         startActivityForResult(intent, 100);
     }
-}
\ No newline at end of file
+
+    // creates menu
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu)
+    {
+        getMenuInflater().inflate(R.menu.menu,menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    // Function is responsible for
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item)
+    {
+        Intent i;
+        switch (item.getItemId())
```

Comment:
I guess this function is not finished, so maybe add little "TODO" comment here, to make it easier to find in the future.

Reply:
there is no menu option, so there is no case in this switch case. But i will add comment and TODO.